units/host_monitoring_idle_termination.go

```diff
@@ -101,7 +101,7 @@ func (j *idleHostJob) Run(ctx context.Context) {
 			return
 		}
 		for _, h := range hosts {
-			j.AddError(errors.Wrapf(h.SetDecommissioned(ctx, evergreen.User, false, "host's distro not found"), "could not set host '%s' as decommissioned", h.Id))
+			j.AddError(errors.Wrapf(h.SetDecommissioned(ctx, evergreen.User, true, "host's distro not found"), "could not set host '%s' as decommissioned", h.Id))
```

Comment:
Looks like this should be `false` since we want it to decommission only if it's idle.

Reply:
done

Reply:
Looks like this one got missed? The other spot got changed to false, but this should also avoid decommissioning the host if it can.

Reply:
sorry I must not have saved this file